client/src/components/Register.js

```diff
@@ -77,7 +84,15 @@ class Register extends React.Component {
             I am a volunteer
           </label>
           <label className="terms">By clicking Register, you agree to the site
-            <Link to="/terms"> terms</Link>
+            <Link onClick={this.handleOpen}> terms</Link>
+            <Modal
+              isOpen={this.state.open}
+              onRequestClose={this.handleClose}
+              contentLabel="Example Modal"
```

Comment:
The `contentLabel` is for the html `aria-label`. Let's change this to say `"Terms"` or something else that is descriptive of the modal.